content/en/continuous_integration/quality_gates/_index.md

```diff
@@ -0,0 +1,171 @@
+---
+title: Quality Gates
+kind: documentation
+description: Learn how to use Quality Gates in your pipeline.
+is_beta: true
+further_reading:
+- link: "/continuous_integration/tests"
+  tag: "Documentation"
+  text: "Learn about Test Visibility"
+---
+
+{{< site-region region="gov" >}}
+<div class="alert alert-warning">CI Visibility is not available in the selected site ({{< region-param key="dd_site_name" >}}) at this time.</div>
+{{< /site-region >}}
+
+{{< callout url="#" header="false" >}}
+Quality Gates are in beta.
+{{< /callout >}}
+
+Quality Gates allows you to gate your workflows based on signals in Datadog.  You can create two types of rules types: Static Analysis and Tests.
```

Comment:
here we say "types of rules types" - can we just say "You can create two rule types:"

Reply:
I have changed it to "two types of rules"